cloudformation/account_resources.yml

```diff
@@ -954,6 +954,16 @@ Resources:
       SecretString: ChangeMe
       Name: !Sub "${AWS::StackName}-SplunkHECToken"
 
+##################################################
+  # FHIR Facade ECR Repository
+##################################################
+  FHIRFacadeRepo:
+    Type: AWS::ECR::Repository
+    Properties: 
+      RepositoryName: fhir-facade-repo
+      ImageScanningConfiguration: 
+        ScanOnPush: true    
+
 Outputs:
```

Comment:
looking at https://nhsd-jira.digital.nhs.uk/browse/AEA-4210 it has the a/c to include adding ECR for validator as well. Can you add it please